source/SAMRAI/geom/CartesianSideDoubleConservativeLinearRefine.C

```diff
@@ -13,11 +13,18 @@
 #include "SAMRAI/hier/Index.h"
 #include "SAMRAI/pdat/SideData.h"
 #include "SAMRAI/pdat/SideVariable.h"
+#include "SAMRAI/pdat/CellData.h"
 #include "SAMRAI/tbox/Utilities.h"
 
 #include <cfloat>
 #include <cmath>
 
+#define MAX(a, b) (((b) > (a)) ? (b) : (a))
+#define MIN(a, b) (((b) < (a)) ? (b) : (a))
+#define ABS abs
+#define SQRT sqrt
+#define COPYSIGN copysign
+
```

Comment:
I think some of these #defines were in other files before you started working on this stuff, but they do make me afraid of conflicts when SAMRAI is linked with other codes.  We should be defining these once in a header somewhere and give them more unique names.  We have utility functions for these types of operations but don't want to call them inside the kernels.  @jonesholger don't worry about it now--I'll suggest something soon.